crates/re_space_view/src/data_query_blueprint.rs

```diff
@@ -467,6 +470,20 @@ impl DataQueryPropertyResolver<'_> {
                         .individual_override_root
                         .join(&node.data_result.entity_path);
 
+                    // If the user has overridden the visualizers, update which visualizers are used.
+                    {
```

Comment:
mesmerizing how little code does the actual lifting of filtering out visualizers :)
should we check against opf --no-frames if this extra query does us any perf harm?